Foundation/NSNotification.swift

```diff
@@ -188,11 +188,6 @@ open class NotificationCenter: NSObject {
         })
     }
 
-    @available(*,obsoleted:4.0,renamed:"addObserver(forName:object:queue:using:)")
```

Comment:
Instead of deleting this, does changing 'obsoleted' to 'unavailable' fix the problem?